doc/manual/en/config/widgets/pushbutton/index.rst

```diff
@@ -40,7 +40,7 @@ Allowed attributes in the PushButton-element
     :align: center
 
     <caption>Attributes in the editor (simple view) [#f1]_</caption>
-    <pushbutton>
+    <pushbutton downValue="1" upValue="0">
```

Comment:
Well, for me this order feels more "natural", because the downValue is the first value to be send, when the use presses the button and the upValue is send after that. But anyways this code only exists to generate the editor screenshot. It isn't even visible in the generated documenation, so the order is irrelevant.